.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```
@@ -0,0 +1,86 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
+
+        Before posting, please check that the bug hasn't already been:
+        * fixed in the next release (https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
+        * discussed previously (https://github.com/stylelint/stylelint/search)
```

```suggestion
        * [fixed in the next release](https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
        * [discussed previously](https://github.com/stylelint/stylelint/search)
```

Let's use markdown links to clean this up a little.